test/maps/azalias/zz_generated_alias_client.go

```diff
@@ -129,6 +135,11 @@ func (client *aliasClient) listCreateRequest(ctx context.Context, options *Alias
 	}
 	reqQP := req.Raw().URL.Query()
 	reqQP.Set("api-version", "2.0")
+	if options != nil && options.GroupBy != nil {
+		for _, qv := range options.GroupBy {
+			reqQP.Add("groupBy", string(qv))
```

Comment:
This is an example of the fix.  Before this you'd get `reqQP.Add("groupBy", qv)` which wouldn't compile.